src/QueryStringHash.ts

```diff
@@ -28,15 +28,13 @@ export function verifyQueryStringHash({
   }
 
   // Check context QSH
-  if (
-    (queryStringHashType === 'context' || queryStringHashType === 'any') &&
-    connectJwt.qsh === ContextQsh
-  ) {
+  const isContext = queryStringHashType === 'context';
+  if ((isContext || queryStringHashType === 'any') && connectJwt.qsh === ContextQsh) {
     return;
   }
 
   // Check computed hash
-  let requestComputedQsh = 'skipped';
+  let requestComputedQsh = isContext ? 'context' : 'skipped';
```

Comment:
This leads to a small improvement on the AuthError object. It now displays `context` on the "Expected" value instead of `skipped`.

This will probably be reviewed in a next PR as I will probably remove the `skip` option from `queryStringHashType` altogether.